app/invocation/invocation.tsx

```diff
@@ -86,7 +97,7 @@ export default class InvocationComponent extends React.Component<Props, State> {
     this.logsSubscription = this.logsModel.onChange.subscribe({
       next: () => this.forceUpdate(),
     });
-    if (!this.isQueued()) {
+    if (!this.isQueued() && !this.props.search.get("runnerFailed")) {
```

Comment:
Would it make sense to show a retry button if this `runnerFailed` prop is set?

Reply:
It will be a bit involved because the current retry button uses BES metadata in order to retry the workflow. We could potentially change the implementation to allow passing in an `Action` to be retried (we would just re-fetch the Github token).